src/components/vue-form-renderer.vue

```diff
@@ -30,7 +30,7 @@
           :name="element.config.name !== undefined ? element.config.name : null"
           v-bind="element.config"
           :is="element.component"
-          :disabled="element.config.interactive"
+          :disabled="element.config.readonly || element.config.interactive"
```

Comment:
I'm not sure we want to mark `readonly` inputs as `disabled`. If an input is disabled, a user cannot interact with it at all, and **the input is not submitted**. This behaviour could be unexpected. If we want to actually disable the input, then we should change the wording in the inspector to reflect that.

MDN describes the readonly attribute here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#readonly.

Reply:
You are right, this change was removed. But actually there is a problem with controls like select, checkbox, etc. where the readonly prop does not block the end user to change the value of the control.